File: Square or Rectangle.java
import java.util.Scanner;

public class Hello {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int length = sc.nextInt();
        int breadth = sc.nextInt();

        if (length == breadth) {
            System.out.println("SQUARE");
        } else {
            System.out.println("RECTANGLE");
        }

    }

}

File: square or rectangle.c
#include<stdio.h>
#include <stdlib.h>

int main()
{
    int length,breadth;
    scanf("%d%d",&length,&breadth);

    if(length == breadth){
        printf("SQUARE");
    }else
    {
        printf("RECTANGLE");
    }

}
